From 26252d83dbee6e65f2e3040ee9c3522cdca9bf4e Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sun, 14 Jul 2019 21:51:30 +0200
Subject: [PATCH 1/1] net: assign maintainer for include/net.h

include/net.h currently has no maintainer.

Assign include/net.h to the maintainer of the NET subsystem.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 MAINTAINERS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index e91684191f..ee93078451 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -614,6 +614,7 @@ M:	Joe Hershberger <joe.hershberger@ni.com>
 S:	Maintained
 T:	git https://gitlab.denx.de/u-boot/custodians/u-boot-net.git
 F:	drivers/net/
+F:	include/net.h
 F:	net/
 
 NIOS
-- 
2.22.0

